src/arithmetic.cpp

```diff
@@ -1796,7 +1507,7 @@ extern "C" blsmul_func_t __attribute__((no_sanitize_address)) resolve_blsmul() {
     return __mul;
```

Comment:
`__mul` not found. Renamed to `__multiply`?

Reply:
fixed